Lib/ctypes/test/test_buffers.py

```diff
@@ -60,5 +60,13 @@ def test_unicode_conversion(self):
         self.assertEqual(b[::2], "ac")
         self.assertEqual(b[::5], "a")
 
+    @need_symbol('c_wchar')
+    def test_create_unicode_buffer_non_bmp(self):
+        b = create_unicode_buffer('\U00010000\U00100000')
```

Comment:
I suggest to test also the largest Unicode code point (currently):

'\U00010000\U0010ffff'